projects/step-core/src/lib/components/predefined-options-input/predefined-options-input.component.html

```diff
@@ -4,7 +4,8 @@
   <input
     [ngModel]="value"
     (ngModelChange)="onValueChange($event)"
-    [disabled]="isDisabled"
+    [disabled]="true"
+    [class.hide-disabled]="!isDisabled"
```

Comment:
If the idea was to make the input non editable, and make it grey only for the disabled state, I think this is the better variant
```suggestion
    [disabled]="isDisabled"
    [readonly]="true"
```